python/jiminy_py/src/jiminy_py/log.py

```diff
@@ -39,3 +41,47 @@ def extract_state_from_simulation_log(log_data, robot):
     return {'evolution_robot': evolution_robot,
             'robot': robot,
             'use_theoretical_model': False}
+
+
+def read_log(filename:str) -> tp.Tuple[tp.Dict, tp.Dict]:
+    """
+    Read a logfile from jiminy. This function supports both text (csv)
+    and binary log.
+
+    Parameters:
+        - filename: Name of the file to load.
+    Retunrs:
+        - A dictionnary containing the logged values, and a dictionnary
+        containing the constants.
+    """
+    if is_binary(filename):
+        # Read binary file using C++ parser.
+        data_dict, constants_dict = Engine.read_log_binary(filename)
+    else:
+        # Read text csv file.
+        constants_dict = {}
+        with open(filename, 'r') as log:
+            consts = next(log).split(', ')
+            for c in consts:
+                c_split = c.split('=')
+                # Remove line end for last constant.
+                constants_dict[c_split[0]] = c_split[1].strip('\n')
+        # Read data from the log file, skipping the first line (the constants).
+        data = read_csv(filename, skiprows=1).to_dict(orient='list')
+        # Convert every element to array to provide same API as the C++ parser,
+        # removing spaces present before the keys.
+        data_dict = {k.strip() : np.array(v) for k,v in data.items()}
+    return data_dict, constants_dict
+
+def is_binary(filename):
```

Comment:
`is_binrary_log` would be better, since it is the only recommended use case of this method (which lack of robustness otherwise)